queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/Repositories/CountrySpecificQueueRepository.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Threading.Tasks;
+using fiskaltrust.Middleware.Contracts.Repositories;
+using fiskaltrust.Middleware.Localization.QueueDEFAULT.Dummy;
+using fiskaltrust.storage.V0;
+
+namespace fiskaltrust.Middleware.Localization.QueueDEFAULT.Repositories
+{
+    public class CountrySpecificQueueRepository : ICountrySpecificQueueRepository
+    {
+        private readonly IConfigurationRepository _configurationRepository;
+
+        public CountrySpecificQueueRepository(IConfigurationRepository configurationRepository)
+        {
+            _configurationRepository = configurationRepository;
+        }
+
+        public Task<ICountrySpecificQueue> GetQueueAsync(Guid queueId) => Task.FromResult<ICountrySpecificQueue>(new CountryDefaultQueue());
+
+        public Task InsertOrUpdateQueueAsync(ICountrySpecificQueue countrySpecificQueue) => Task.CompletedTask;
```

Comment:
This means that we will never increase the counters that we have right? Also FailedCounters are usually set in the countryspecific queue. Is this something that we should look out for?

Reply:
The thing is that we don't have these counters because we don't have a default countryspecific queue because we did not want to add default stuff to the storage. Since we also don't sign in the default queue we don't have failed counters 🤔 

BTW this PR is for now only the default queue and in the next step we will add comments and more dummy code to make it the new market template. then we will add explanations and todos for all of that.